base/applications/osk/lang/de-DE.rc

```diff
@@ -238,69 +238,69 @@ BEGIN
-    CTEXT           "Scroll",IDC_STATIC,414,3,21,8
+    CTEXT           "Rollen",IDC_STATIC,414,3,21,8
     CONTROL         "",IDC_LED_SCROLL,"Static",SS_CENTER|SS_NOTIFY,424,15,4,3
 END
 
 IDR_OSK_MENU MENU DISCARDABLE
 BEGIN
-    POPUP "File"
+    POPUP "Datei"
     BEGIN
-        MENUITEM "&Exit", IDM_EXIT
+        MENUITEM "&Beenden", IDM_EXIT
     END
 
-    POPUP "Keyboard"
+    POPUP "Tastatur"
     BEGIN
-        MENUITEM "Enhanced Keyboard", IDM_ENHANCED_KB, CHECKED
-        MENUITEM "Standard Keyboard", IDM_STANDARD_KB
+        MENUITEM "Erweiterte Tastatur", IDM_ENHANCED_KB, CHECKED
+        MENUITEM "Standard Tastatur", IDM_STANDARD_KB
         MENUITEM SEPARATOR
-        MENUITEM "Regular Layout", IDM_REG_LAYOUT, CHECKED, GRAYED
+        MENUITEM "Reguläres Layout", IDM_REG_LAYOUT, CHECKED, GRAYED
         MENUITEM "Block Layout", IDM_BLOCK_LAYOUT, GRAYED
         MENUITEM SEPARATOR
-        MENUITEM "101 keys", IDM_101_KEYS, CHECKED, GRAYED
-        MENUITEM "102 keys", IDM_102_KEYS, GRAYED
-        MENUITEM "106 keys", IDM_106_KEYS, GRAYED
+        MENUITEM "101 Tasten", IDM_101_KEYS, CHECKED, GRAYED
+        MENUITEM "102 Tasten", IDM_102_KEYS, GRAYED
+        MENUITEM "106 Tasten", IDM_106_KEYS, GRAYED
     END
 
-    POPUP "Settings"
+    POPUP "Einstellungen"
     BEGIN
-        MENUITEM "Always on Top", IDM_ON_TOP, CHECKED
+        MENUITEM "Immer im Vordergrund", IDM_ON_TOP, CHECKED
         MENUITEM SEPARATOR
-        MENUITEM "&Use Click Sound", IDM_CLICK_SOUND
+        MENUITEM "&Benutze Klick-ton", IDM_CLICK_SOUND
         MENUITEM SEPARATOR
         MENUITEM "&Typing Mode...", IDM_TYPE_MODE, GRAYED
         MENUITEM SEPARATOR
-        MENUITEM "&Font...", IDM_FONT, GRAYED
+        MENUITEM "&Schiftart...", IDM_FONT, GRAYED
     END
 
-    POPUP "Help"
+    POPUP "Hilfe"
     BEGIN
-        MENUITEM "&Help Topics", IDM_HELP_TOPICS, GRAYED
+        MENUITEM "&Hilfethemen", IDM_HELP_TOPICS, GRAYED
         MENUITEM SEPARATOR
-        MENUITEM "&About On-Screen Keyboard...", IDM_ABOUT
+        MENUITEM "&Über Bildschirmtastatur...", IDM_ABOUT
     END
 END
 
 IDD_WARNINGDIALOG_OSK DIALOGEX 0, 0, 250, 97
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUP | WS_CAPTION
-CAPTION "On-Screen Keyboard"
+CAPTION "Bildschirmtastatur"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
     DEFPUSHBUTTON "OK", IDOK, 193, 76, 50, 14
     ICON IDI_OSK, IDC_STATIC, 7, 17, 20, 20
-    LTEXT "On-Screen Keyboard provides a minimum level of functionality for mobility-impaired users. Mobility-impaired users will need a utility program with higher functionality for daily use.", IDC_STATIC, 36, 7, 207, 33
-    CONTROL "Do not show this message again", IDC_SHOWWARNINGCHECK, "Button",
+    LTEXT "Bildschirmtastatur stellt ein mindestmaß an Funktionalität für verhinderte Nutzer zur Verfügung. Solche Benutzer werden ein Program mit größerem Funktionsumfang brauchen für den täglichen Gebrauch.", IDC_STATIC, 36, 7, 207, 33
```

Comment:
Eingeschränkt instead of verhindert.

Diese Nutzer werden, für den täglichen Gebrauch,... then the rest of the sentence